### PR TITLE
chore: Improve error message if maker is unable to open channel

### PR DESCRIPTION
**Pull request by klochowicz**
Match on all errors, and make it clear that the error is from the maker.

### CODE REVIEW

File: rust/src/lightning.rs [hunk 1]
```diff
@@ -183,8 +183,7 @@ pub async fn open_channel(
             .map_err(|e| anyhow!("Could not create channel with {peer_info} due to {e:?}"))?,
     };
 
-    tracing::debug!("Created channel with {peer_info}");
-    tracing::info!("Channel has been successfully created");
+    tracing::info!("Created channel with {peer_info}");
```
Comment (luckysori) [inline rust/src/lightning.rs:186]: 🔧 Not a problem with your PR, but the message has always been misleading.

```suggestion
    tracing::info!("Started channel creation with {peer_info}");
```

**Review by luckysori — APPROVED** [review]

**Comment by luckysori** [comment]
@klochowicz: Can we close #374 with this PR?

**Comment by klochowicz** [comment]
> @klochowicz: Can we close #374 with this PR?

I have already linked it in 'development' when opening the PR, so I assume it will be closed automatically ;-)

File: rust/src/lightning.rs (hunk 1 above)
Comment (klochowicz) [reply rust/src/lightning.rs:186]: fixed :)

**Comment by luckysori** [comment]
> I have already linked it in 'development' when opening the PR, so I assume it will be closed automatically ;-)

Oops, missed that.

**Pull request merged**